### PR TITLE
Create a walkthrough experience for PowerShell

### PR DESCRIPTION
**Pull request by S-Hakim**
Adds a walkthrough to the extension! Resolves https://github.com/PowerShell/vscode-powershell/issues/4080.

### CODE REVIEW

**Review by SydneyhSmith — APPROVED** [review]

**Review by andschwa — APPROVED** [review]
I made a few edits but I love this!

File: package.json [hunk 1]
```diff
@@ -897,6 +898,77 @@
...
+        "id": "PowerShell",
+        "title": "Get Started with PowerShell",
+        "description": "Set up your environment and see what the PowerShell Extension in Visual Studio Code can help you achieve!",
+        "steps": [
+          {
+            "id": "version",
+            "title": "Choose a version of PowerShell",
+            "media": {
+              "markdown": "media/walkthrough.md"
+            },
```
Comment (andschwa) [inline package.json:913]: Note: this empty media key is annoyingly required for the walkthrough to show up, even though it does nothing.

Comment (TylerLeonhardt) [reply package.json:913]: Can you open an issue about this?

Comment (andschwa) [reply package.json:913]: I'm _sure_ it's known already!

**Pull request merged**